frontend/src/app/modules/boards/board/board.service.ts

```diff
@@ -20,6 +20,8 @@ export interface CreateBoardParams {
 @Injectable({ providedIn: 'root' })
 export class BoardService {
 
+  public currentBoard$:BehaviorSubject<string> = new BehaviorSubject<string>('');
```

Comment:
@HDinger I would use a null instead of an empty string to indicate the absence of value. All the rest LGTM